### PR TITLE
Send an email after creating an account

### PR DESCRIPTION
**Pull request by sebastiancaraballo**
**Trello board reference:**

- https://trello.com/c/az8rNczQ

**Description:**
- As a user I should be able to receive an email after signing up.

### CODE REVIEW

File: app/mailers/user_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class UserMailer < ApplicationMailer
+  default from: 'no-reply@api.com'
+
+  def welcome_email
+    @user = params[:user]
+    @url = new_user_session_url
+    mail(to: @user.email, subject: I18n.t('api.emails.welcome_message.subject'))
```
Comment (gabrielbursztein2) [inline app/mailers/user_mailer.rb]: Just `t('api.emails.welcome_message.subject')` should work.

**Review by gabrielbursztein2 — APPROVED** [review]
Really nice work! Just a tiny comment.

**Review by jpascual1994 — APPROVED** [review]

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -66,4 +66,13 @@
     }
     it { should have_many(:conversations).through(:user_conversations) }
   end
+
+  describe 'callbacks' do
+    it { is_expected.to callback(:send_welcome_mail).after(:create) }
+
+    it 'adds email to queue' do
+      ActiveJob::Base.queue_adapter = :test
```
Comment (mdornel) [inline spec/models/user_spec.rb]: Move this configuration setting to `spec_helper.rb`

**Review by mdornel — COMMENTED** [review]
Great work! just a minor comment

**Pull request merged**